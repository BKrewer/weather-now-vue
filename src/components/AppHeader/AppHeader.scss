.app-header {
    background-color: $white;
    box-shadow: 0px 3px 5px 0px $box-shadow;
    display: flex;
    height: 70px;
    justify-content: center;
    width: 100%;

    &__logo {
        max-width: 200px;
        width: 100%;
    }
}