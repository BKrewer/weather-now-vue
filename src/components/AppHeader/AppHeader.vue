<template>
  <header class="app-header">
      <img class="app-header__logo" alt="Weathernow" src="@/assets/images/logo.svg">
  </header>
</template>

<style lang="scss" src="./AppHeader.scss" />