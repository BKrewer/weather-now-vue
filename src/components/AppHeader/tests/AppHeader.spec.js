import { shallowMount } from '@vue/test-utils'
import AppHeader from '@/components/AppHeader/AppHeader.vue'

describe('AppHeader.vue', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(AppHeader)
    expect(wrapper.element).toMatchSnapshot();
  })
})