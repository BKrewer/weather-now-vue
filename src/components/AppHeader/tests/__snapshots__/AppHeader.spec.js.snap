// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader.vue renders correctly 1`] = `
<header
  class="app-header"
>
  <img
    alt="Weathernow"
    class="app-header__logo"
    src=""
  />
</header>
`;
